Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.7.2
+
+RUN apt-get update -qq && apt-get upgrade -y
+RUN apt-get install -y build-essential nodejs && apt-get clean
+RUN gem install foreman
+
+ENV GOVUK_APP_NAME authenticating-proxy
+ENV PORT 3107
```

Comment:
This duplicates the default that you've defined in the Procfile and makes it a bit harder to understand how to override the port. Ideally there should be no need to define this env var in the Dockerfile.